specs/interop/token-bridging.md

```diff
@@ -24,60 +18,55 @@
 ## Overview
 
 Without a standardized security model, bridged assets may not be fungible with each other.
-The `SuperchainERC20` unifies ERC20 token bridging to make it fungible across the Superchain.
-It builds on top of the messaging protocol, as the most trust minimized bridging solution.
+The `SuperchainERC20` standard is a set of properties allowing ERC20 to be fungible across the
+Superchain using the official `SuperchainERC20Bridge`.
+The `SuperchainERC20Bridge` is a predeploy that builds on the messaging protocol as the most trust-minimized bridging solution.
 
-Unlike other standards, such as [xERC20](https://www.xerc20.com/),
-where users interact with a bridge possessing mint and burn
-privileges on cross-chain enabled tokens, this approach allows users to call methods directly on the token contract.
+## `SuperchainERC20` standard
 
-## Interface
+The standard will build on top of ERC20 and include the following properties:
 
-### Functions
+1. Give `mint` and `burn` rights to the `SuperchainERC20Bridge`.
```

Comment:
Is it possible to make the mint / burn requirements wrt rate limits more explicit (for clarity around xerc20-superchain implementations)? 

Specifically, I think it would help if this states whether infinite mint/burn permissions are necessary or if rate limited as per the vanilla xerc20 implementation is fine. If it doesn't matter then I think this should still be stated for clarity.

Reply:
I don't think that we should enshrine any particular logic in the bridge about rate limiting. There isn't a universal way of doing rate limits that will solve every usecase, so any rate limits should be done in the token contracts themselves. It is a good callout that the spec could mention rate limiting implemented in the token or another contract

Reply:
Yeah that's fine. Just wanted to make it crystal clear whether a rate limited erc20 was compatible with superchainerc20 or not. And it sounds like the answer is that it is compatible.

Reply:
it should be, yes, and let's say the rate limit is exceeded, you'd have 2 options.
1) allow for replaying the message when the rate limit enables it.
2) expire the message and trigger a "rollback" interop message, which would re-mint the funds back on origin.